README.MD

```diff
@@ -55,23 +61,75 @@ Only basic features that almost all projects use, were added in this template:
-deploy builds to Firebase App Distribution and Play Store.
+If you need local signing of release builds, copy the upload keystore into the root of your project,
+and call it 'upload-keystore.jks'. Note that you probably don't need to use local signing. The CI
+build setup builds and signs your app for
+you.
 
-#### Installing fastlane
+### CI / Automated Builds
 
-A big advantage of fastlane is that the ci and publishing steps can run locally.
-Use `brew install fastlane` and then `fastlane test` to verify this installation.
+This project uses Github Actions for CI. We have added these workflows for now:
 
-#### Removing Github Actions + fastlane
+- _debug.yml_: any commit on any branch triggers a `devDebug` apk and a `prodDebug` apk build
+- _release.yml_: any PR triggers signed release builds (`prodRelease` bundle and `prodRelease` apk)
 
-Using Bitrise instead? Then you can delete:
+#### Adding your own keystore's details on Github Actions
+
+- First create your own keystore and store it in 1pw. _You do not add this keystore to the repo_ (
+  like we did in this template as an example).
+- add a KEYSTORE_BASE_64 [new github repository secret](./settings/secrets/actions/new) that is the
+  encoded text
+  representation of the upload
+  keystore of your app. CI will later decode it into a keystore file and then make it available for
+  the system to sign the app.
+
+Run the below command to generate an encoded text representation
+of your keystore. If you don't have one, you
+can [generate a new keystore](https://developer.android.com/studio/publish/app-signing#generate-key).
+
+    openssl base64 < upload-keystore.jks | tr -d '\n' | tee keystore.base64.txt
+
+The above command will generate a new file called `keystore.base64.txt`. Open the file, copy the
+contents and save it to your repo's github secrets in the variable KEYSTORE_BASE_64.
+
+Also [add these repository secrets](./settings/secrets/actions/new) in github, and store them in
+your projects 1Password vault as well:
+
+- RELEASE_KEYSTORE_PASSWORD ('firstpass' for this template)
+- RELEASE_KEYSTORE_ALIAS ('template' for this template)
+- RELEASE_KEY_PASSWORD ('secondpass' for this template)
+
+#### Firebase app distribution
+
+We use a [firebase app github action](https://github.com/nickwph/firebase-app-distribution-action) (
+in our github actions setup) to automatically upload release builds to firebase.
+
+To change this to your own firebase project, you need to set two secrets:
+
+- `FIREBASE_PROD_APP_ID` from your firebase project settings. It's in your firebase project
+  settings (for the app you want to publish), it looks something like `1:xxxxxxxxx:android:yyyyyyyy`
+- `FIREBASE_CREDENTIALS` from your firebase project. More info how to obtain it can be found
+  in [creating a service account](https://github.com/nickwph/firebase-app-distribution-action?tab=readme-ov-file#download-credentials-from-firebase).
+
+#### Self-hosted runner
```

Comment:
https://www.notion.so/q42/GitHub-Actions-CI-op-onze-self-hosted-runner-Mac-Mini-7f7d2e6312444b98be521394223db6e2?pvs=4#99626170e0e24fc1ae48d30f88d96b59

Unsure whether this could work outside q42 org

Reply:
Probably not, you can then let the customer pay :)